code/game/gamemodes/bloodsuckers/traitorsuckers.dm

```diff
@@ -0,0 +1,63 @@
+/datum/game_mode/traitor/bloodsucker
+	name = "traitor+bloodsucker"
+	config_tag = "traitorsucker"
+	report_type = "traitorsucker"
+	false_report_weight = 10
+	traitors_possible = 3 // Hard limit on Traitors if scaling is turned off
+	restricted_jobs = list("AI", "Cyborg")
+	protected_jobs = list("Captain", "Head of Personnel", "Head of Security", "Research Director", "Chief Engineer", "Chief Medical Officer", "Curator", "Warden", "Security Officer", "Detective", "Brig Physician")
+	required_players = 20
+	required_enemies = 1 // How many of each type are required
+	recommended_enemies = 2
+	reroll_friendly = 1
+	announce_span = "Traitors and Bloodsuckers"
+	announce_text = "There are vampiric monsters on the station along with some syndicate operatives out for their own gain! Do not let the bloodsuckers or the traitors succeed!"
+
+	var/list/possible_bloodsuckers = list()
+	var/const/bloodsucker_amount = 2
+
+/datum/game_mode/traitor/bloodsucker/can_start()
+	. = ..()
+	if(!.)
+		return
+	possible_bloodsuckers = get_players_for_role(ROLE_BLOODSUCKER)
+	if(possible_bloodsuckers.len < required_enemies)
+		return FALSE
+	return TRUE
+
+/datum/game_mode/traitor/bloodsucker/pre_setup()
+	if(CONFIG_GET(flag/protect_roles_from_antagonist))
+		restricted_jobs += protected_jobs
+
+	if(CONFIG_GET(flag/protect_assistant_from_antagonist))
+		restricted_jobs += "Assistant"
+
+	var/list/datum/mind/possible_bloodsuckers = get_players_for_role(ROLE_BLOODSUCKER)
```

Comment:
either this list or the gamemode list of the same name needs to have its name changed